Assets/Scripts/PlayerInteract.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class PlayerInteract : MonoBehaviour
+{
+    public bool interact = false;
```

Comment:
```suggestion
    [HideInInspector]
    public bool interact = false;
```
...zorgt ervoor dat een public variable niet in de inspector wordt laten zien